src/app/octoprint-api/filesAPI.ts

```diff
@@ -14,6 +14,7 @@ export interface OctoprintFilesAPI {
     statistics?: object;
     type: string;
     typePath: [string];
+    thumbnail?: [string];
```

Comment:
Is this really an array?

Reply:
No, I just wasn't paying attention...which is why my other calls were causing errors hence the concatentation...